test/node/test-buffer-from.js

```diff
@@ -66,3 +67,73 @@ deepStrictEqual(
   throws(() => Buffer.from(input), errMsg);
 });
 
+{
+  const u16 = new Uint16Array([0xffff]);
+  const b16 = Buffer.copyBytesFrom(u16);
+  u16[0] = 0;
+  assert.strictEqual(b16.length, 2);
+  assert.strictEqual(b16[0], 255);
+  assert.strictEqual(b16[1], 255);
```

Comment:
How are semi's here? :thinking:
Shouldn't the linter break on this?

Reply:
(we can resolve this in another pull request)

Reply:
I think it's because I pulled these directly from the node.js tests.

Reply:
If these are directly from the node tests, can you add an please add a reference?
(and potentially adhere to whatever LICENSE notice may be required there)

Reply:
Is that not already accounted for? As far as I can tell, all of the tests in `test/node` are ripped straight from the node.js codebase, right down to the filenames. I even recognize [these ones][1] -- I wrote them. :blush:

[1]: https://github.com/feross/buffer/blob/master/test/node/test-buffer-badhex.js

Reply:
Thanks for hunting that down @chjj, both are MIT, so hopefully this is simply about attribution, I think an inline URL would be helpful too so others can verify the tests against the upstream